mod/01_core/set/all/content.rb

```diff
@@ -19,8 +19,11 @@ def raw_content
   structure ? template.db_content : db_content
 end
 
-def chunk_list #override to customize by set
-  :default
+
+format :all do
+  def chunk_list #override to customize by set
+    :default
+  end
```

Comment:
98     def include_set_format_modules
 99       self.class.format_ancestry.reverse.each do |klass|
100         card.set_format_modules( klass ).each do |m|
101           singleton_class.send :include, m
102         end
103       end
104     end

Seems to me that this handles the case I was talking about.  The loop on 99 will load set modules from the inheritance classes with the right order so inheriting class' methods are overloaded on top to override methods from the classes up the line.  Add the tests and see if you still need this stuff.  There may be a use case when you want something in two formatters where one isn't an ancestor of the other, but I'd bet that is very rare.